src/eventtimes.jl

```diff
@@ -39,3 +39,9 @@ end
 
 isevent(ev::EventTime) = ev.status
 iscensored(ev::EventTime) = !ev.status
+
+## StatsModels compatibility
+
+StatsModels.concrete_term(t::Term, xs::AbstractVector{<:EventTime}, ::Nothing) =
+    StatsModels.ContinuousTerm(t.sym, first(xs), first(xs), first(xs), first(xs))
+Base.copy(et::EventTime) = et
```

Comment:
Shouldn't this actually copy? Like `EventTime(copy(et.time), copy(et.status))`

Reply:
I guess that's a no-op though since numbers and `Bool`s are immutable. Though I guess the time could be a `BigFloat`, for which `copy` actually does something (I think?).

Reply:
I dunno I was lazy

Reply:
It's immutable though right?

Reply:
I mean it is up 2 u

Reply:
Nah, this is fine.